### PR TITLE
Redate MBC2 results news post

### PR DESCRIPTION
**Pull request by pishifat**
i blame noffy

### CODE REVIEW

File: news/2020-03-20-monthly-beatmapping-contest-2-results.md [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
 
 As the contest's runner-up, UndeadCapulet earned a month of supporter and is halfway to the first reward tier's profile badge prize! See the rewards breakdown on the [MBC wiki article's rewards section](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest#rewards) for more info about how this works.
 
-**Screening crew's favorite:** **[Voicians - Wolves](https://osu.ppy.sh/beatmapsets/1129015#osu/2358657)** by **[Mao](https://osu.ppy.sh/users/2204515)**, masked as Conscientious Elephant.
+---
 
-Submissions to the MBC are screened by about 10 experienced beatmappers to decide the finalists. This month, Mao's entry placed in the top 5 for 2/3 of those mappers, far surpassing any other entry in terms of screening votes!
+Submissions to this month's MBC were screened by 14 experienced beatmappers to decide the finalists. This month, [Mao](https://osu.ppy.sh/users/2204515)'s entry for (yet again) [Voicians - Wolves](https://osu.ppy.sh/beatmapsets/1129015#osu/2358657) placed in the top 5 for 9 of those mappers, far surpassing any other entry in terms of screening votes and earnings its spot as the **screening crew's favorite!**
```
Comment (cl8n) [inline news/2020-03-20-monthly-beatmapping-contest-2-results.md]: this link is to beatmap page but the others are a download from contests page, idk if it's really a problem that they differ i just wouldn't expect

File: news/2020-03-20-monthly-beatmapping-contest-2-results.md [hunk 2]
```diff
@@ -1,40 +1,38 @@
 ---
 layout: post
-title: "Monthly Beatmapping Contest #1: Results"
-date: 2020-03-21 23:00:00 +0000
+title: "Monthly Beatmapping Contest #2: Results"
+date: 2020-03-20 23:00:00 +0000
```
Comment (cl8n) [inline news/2020-03-20-monthly-beatmapping-contest-2-results.md]: at this point you could do 21 0:00

File: news/2020-03-20-monthly-beatmapping-contest-2-results.md (hunk 1 above)
Comment (pishifat) [reply news/2020-03-20-monthly-beatmapping-contest-2-results.md]: id prioritize osuweb links over contest links, others havent uploaded though

File: wiki/Contests/Monthly_Beatmapping_Contest/en.md [hunk 3]
```diff
@@ -51,8 +51,8 @@ Information and results for all iterations of the MBC since its relaunch in 2020
 | Contest Listing | Winner | Runner-up | Screening crew's #1 | Results Post |
 | :-- | :-- | :-- | :-- | :-- |
 | [January 2020](https://osu.ppy.sh/community/contests/84) | ![][flag_DK] [melon boy](https://osu.ppy.sh/users/3053382) | ![][flag_CN] [Moecho](https://osu.ppy.sh/users/5075660) | ![][flag_GB] [Altai](https://osu.ppy.sh/users/5745865) | [Results](https://osu.ppy.sh/home/news/2020-02-20-monthly-beatmapping-contest-1-results) |
-| [February 2020](https://osu.ppy.sh/community/contests/89) | ![][flag_RU] [Aphestra](https://osu.ppy.sh/users/11949191) | ![][flag_US] [UndeadCapulet](https://osu.ppy.sh/users/2523533) | ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | [Results](https://osu.ppy.sh/home/news/2020-03-22-monthly-beatmapping-contest-2-results) |
- [March 2020](https://osu.ppy.sh/community/contests/91) | TBD | TBD | TBD | TBD |
+| [February 2020](https://osu.ppy.sh/community/contests/89) | ![][flag_RU] [Aphestra](https://osu.ppy.sh/users/11949191) | ![][flag_US] [UndeadCapulet](https://osu.ppy.sh/users/2523533) | ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | [Results](https://osu.ppy.sh/home/news/2020-03-20-monthly-beatmapping-contest-2-results) |
```
Comment (cl8n) [inline wiki/Contests/Monthly_Beatmapping_Contest/en.md]: u meme

**Pull request merged**